### DISCUSSION TITLE
Why can't we intercept the route?

### DISCUSSION THREAD

**Discussion by bbhxwl** [vercel/next.js#69844] [App Router]
Why can't we intercept the route?

<img width="1093" alt="image" src="https://github.com/user-attachments/assets/58969c6c-f70f-4caf-a443-20c514cef1c0">
<img width="1181" alt="image" src="https://github.com/user-attachments/assets/5aee8ff6-334e-4e5d-ab3e-2b61fdf09904">
<img width="1104" alt="image" src="https://github.com/user-attachments/assets/b55c03f9-e5c4-4f0e-b30f-5f0b215661b6">

**Comment by romhenri** [comment]
Can you explain better?

**Comment by Choi-Jiwon-38** [comment]
What version of Next.js are you using?
